src/dbspro/cli/run.py

```diff
@@ -15,6 +15,9 @@
 accepted_file = 'reads.fastq.gz'
 accepted_file_ext = '.fastq.gz'
 
+default_handles_file = "construct-info/handles.tsv"
+default_abc_file = "construct-info/ABC-sequences.tsv"
```

Comment:
I think these could be written in the `default` in the argparser.

Reply:
That also means lines 53-57 are not needed.

Reply:
Yes they are removed in last version